### ISSUE TITLE
browser-entry.js not present when mocha 2.5.3 is installed via npm

### ISSUE THREAD

**Issue by qrohlf** [mochajs/mocha#2323]
```sh
cd /tmp; npm i mocha; stat node_modules/browser-entry.js
# => stat: node_modules/browser-entry.js: stat: No such file or directory
```

This causes `require('mocha')` to fail in a webpack environment

**Comment by boneskull** [comment]
@qrohlf I'll see if I can't fix this today

**Comment by qrohlf** [comment]
@boneskull awesome thanks, for now I can probably workaround by overwriting node_modules/mocha with the source from GitHub

**Comment by boneskull** [comment]
[build in progress](https://travis-ci.org/mochajs/mocha/builds/139292580)

**Comment by callmevlad** [comment]
@boneskull Any chance this fix can be published to npm as `mocha@2.5.4`?
